src/libraries/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -92,5 +92,11 @@
     <ProjectReference Include="VoidMainWithExitCodeApp\VoidMainWithExitCodeApp.csproj" />
     <ProjectReference Include="TestApp\TestApp.csproj" />
     <ProjectReference Include="TargetFrameworkNameTestApp\TargetFrameworkNameTestApp.csproj" />
+
+    <TrimmerRootAssembly Include="AssemblyResolveTestApp" />
```

Comment:
Why do we somtimes use a ILLink.Descriptors.xml file to specify assemblies, but here we use `TrimmerRootAssembly`? Should we be consistent everywhere?